### DISCUSSION TITLE
List-Like Dictionary Extension

### DISCUSSION THREAD

**Discussion by BlueDome77** [TurboWarp/extensions#550] [Ideas]
I know there is already a **Dictionary** extension but maybe someone could make one that looks and acts like normal scratch lists. (See concept below that I made &#x23F7; )

![Screenshot 2023-06-09 at 7 27 29 AM](https://github.com/TurboWarp/extensions/assets/135789443/54838a78-e3c2-4d19-b3e2-dd25d442402e)

**Comment by LilyMakesThings** [comment]
Custom monitors are currently unavailable. You can do this with JSON arrays and normal lists.

**Reply by BlueDome77** [reply]
What do you mean by “currently unavailable”?

**Reply by LilyMakesThings** [reply]
We can't make our own monitor types, so this would be impossible.

**Reply by yuri-kiss** [reply]
sad
